tests/lib/config.js

```diff
@@ -884,6 +884,38 @@ describe("Config", function() {
             assertConfigsEqual(actual, expected);
         });
 
+        it("should extend package sub-configuration with prefix", function() {
+
+            var configDeps = {};
+            configDeps["./config/config-file"] = proxyquire("../../lib/config/config-file", {
+                "@scope/eslint-config/foo": {
+                    rules: {
+                        eqeqeq: 2
+                    }
+                }
+            });
+
+            var StubbedConfig = proxyquire("../../lib/config", configDeps);
+
+            var configPath = path.resolve(__dirname, "../fixtures/config-extends/scoped-package9/.eslintrc"),
+                configHelper = new StubbedConfig({ useEslintrc: true, configFile: configPath, cwd: process.cwd() }),
+                expected = {
+                    rules: { "quotes": [2, "double"], "eqeqeq": 2, "valid-jsdoc": 0 },
+                    env: { "browser": false }
+                };
+
+            // Reason to override this function in this special case is that I don't want it to keep looking into the
+            // chain of parent directories for .eslintrc or package.json file for configs.
+            // If that is allowed then the expected outcome will change based on the machine you are running this test.
+            sinon.stub(configHelper, "findLocalConfigFiles").returns([
+                path.resolve(__dirname, "../fixtures/config-extends/scoped-package9/.eslintrc")
```

Comment:
Ah I see that this is duplicated from other tests. I'm guessing support for extending configs was written before we supported the root option. If it's as simple as adding the root option and deleting these stubs, can you update these?